mllib/src/test/scala/org/apache/spark/mllib/feature/ChiSqSelectorSuite.scala

```diff
@@ -189,6 +189,6 @@ object ChiSqSelectorSuite extends SparkFunSuite {
   }
 
   def checkEqual(a: ChiSqSelectorModel, b: ChiSqSelectorModel): Unit = {
-    assert(a.selectedFeatures.deep == b.selectedFeatures.deep)
+    assert(a.selectedFeatures.sameElements(b.selectedFeatures)
```

Comment:
Hi, @srowen . This is a compile error~
```
[error] /Users/dongjoon/PRS/PR-26073/mllib/src/test/scala/org/apache/spark/mllib/feature/ChiSqSelectorSuite.scala:193: ')' expected but '}' found.
[error]   }
[error]   ^
[error] one error found
```

Reply:
Ugh I messed up something translating between my two builds. Will fix